aggregator/handler/config.go

```diff
@@ -197,12 +202,26 @@ func (c *dynamicBackendConfiguration) NewSharderRouter(
 	if err := p.Init(); err != nil {
 		return SharderRouter{}, err
 	}
+	for _, filter := range c.Filters {
+		sid, f := filter.NewConsumerServiceFilter()
+		p.RegisterFilter(sid, f)
+		instrumentOpts.Logger().Infof("registered filter for consumer service: %s", sid.String())
```

Comment:
nit: move the logger outside the for loop.

Also do you need this log message? is it too verbose?

Also do you want to implement `String()` for the filter?

Reply:
It's probably fine since it only print once per each defined filter, usually we will only have one of this, I feel it's nice to have something to indicate the filter is being applied.